Assets/Scripts/TransitionManager.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+public class TransitionManager : MonoBehaviour
+{
+    void Start()
+    {
+        
+    }
+
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.R))
```

Comment:
consider moving `KeyCode.R` into a variable at the top of this file. `resetButton` or something, so we can easily change it or see what it's set to if we forget what it is. saves having to search through entire code files 😄